voltixApp/VoltixApp/Model/DeviceInfo.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//  VoltixApp
+//
+//  Created by Enrique Souza Soares on 10/04/2024.
+//
+
+import Foundation
+import SwiftUI
+
+struct DeviceInfo {
+    var Index: Int
+    var Signer: String
+    var PubKey: String
```

Comment:
can you please remove `PubKey` from `DeviceInfo`

Reply:
Done

Reply:
Done